pkg/flavor/kubernetes/flavor.go

```diff
@@ -574,3 +574,15 @@ func (flavor *Flavor) GetVolumePropertyOfPV(propertyName string, pvName string)
 	}
 	return "", nil
 }
+
+func (flavor *Flavor) GetOrchestratorVersion() (*version.Info, error) {
+	log.Tracef(">>>>> GetOrchestratorVersion")
+	defer log.Tracef("<<<<< GetOrchestratorVersion")
+
+	version, err := flavor.kubeClient.Discovery().ServerVersion()
```

Comment:
nit: maybe name this variable versionInfo. It looks like we have a version object here

Reply:
sure..